roaster-zephyr/roaster-zephyr-common/src/main/java/hu/icellmobilsoft/roaster/zephyr/common/client/RestZephyrService.java

```diff
@@ -45,26 +48,28 @@
  * @author mark.vituska
  * @since 0.11.0
  */
-@ApplicationScoped
 public class RestZephyrService {
 
-    @Inject
-    @RestClient
-    private ZephyrRestClient zephyrClient;
-
-    @Inject
-    @RestClient
-    private JiraRestClient jiraClient;
-
-    @Inject
-    private IJiraReporterServerConfig serverConfig;
-
-    @Inject
-    private IZephyrReporterConfig zephyrConfig;
-
-    private static final Set<String> existingTestCycleKeys = new HashSet<>();
-
-    private static final Map<String, String> accountIdsByUserName = new HashMap<>();
+    private final ZephyrRestClient zephyrClient = RestClientBuilderResolver.instance()
+            .newBuilder()
+            .baseUri(
+                    ConfigProvider.getConfig()
+                            .getOptionalValue("ZephyrRestClient/mp-rest/url", URI.class)
```

Comment:
Shouldn’t this be included in the release notes?
If I'm not mistaken, the config keys changed from hu.icellmobilsoft.roaster.zephyr.common.client.api.ZephyrRestClient/mp-rest/url to ZephyrRestClient/mp-rest/url (probably no one actually overrode it, but still...).